src/matter/cluster/PressureMeasurementCluster.ts

```diff
@@ -45,7 +45,7 @@ export const PressureMeasurementCluster = Cluster({
        maxScaledValue: OptionalAttribute(0x12, TlvNullable(TlvInt16), { default: 0 }),
 
        /** Indicates the magnitude of the possible error that is associated with ScaledValue */
-       scaledTolerance: OptionalAttribute(0x13, TlvUInt16.bound({ min: 0, max: 2048 }), { default: 0 }),
+       scaledTolerance: OptionalAttribute(0x13, TlvUInt16.bound({ min: 0, max: 2048 /* 0x0800 */ }), { default: 0 }),
 
        /** Indicates the base 10 exponent used to obtain ScaledValue */
        scale: OptionalAttribute(0x14, TlvUInt8.bound({ min: -127, max: 127 }), { default: 0 }),
```

Comment:
This is incorrect, TlvUInt8 is an unsigned int so cannot be below 0. Should this be a signed int?